From: Stephen M. Cameron <scameron@beardog.cce.hp.com>
Date: Mon, 14 Jan 2013 12:26:54 +0100
Subject: hpsa: add __devinit annotation to hpsa_wait_for_board_state

References: FATE#313833
Patch-Mainline: n/a
It's already marked that way in the prototype but this makes
code closer to kernel.org codebase.

Signed-off-by: Stephen M. Cameron <scameron@beardog.cce.hp.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/hpsa.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index 7e2f890..83c06f6d 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -4127,7 +4127,7 @@ static int __devinit hpsa_pci_find_memory_BAR(struct pci_dev *pdev,
 	return -ENODEV;
 }
 
-static int hpsa_wait_for_board_state(struct pci_dev *pdev,
+static int __devinit hpsa_wait_for_board_state(struct pci_dev *pdev,
 	void __iomem *vaddr, int wait_for_ready)
 {
 	int i, iterations;
-- 
1.7.4.2

